framework_tvb/tvb/core/entities/file/simulator/view_model.py

```diff
@@ -318,6 +318,22 @@ class SimulatorAdapterModel(ViewModel, Simulator):
     def linked_has_traits(self):
         return Simulator
 
+    #TODO: add get_algorithm_module and get_algorithm_class_name to all modules (for project import)
+
+    @property
+    def get_algorithm_module(self):
```

Comment:
Since this function represents is actually an object property, I would rename it to algorithm_module. Same thing for 'get_algorithm_class_name'